### PR TITLE
A more versatile README

### PR DESCRIPTION
**Pull request by kszucs**
Just a quick skeleton for https://github.com/dask/dask/issues/2230 
rendered here https://github.com/kszucs/dask/tree/readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+x = 1
+y = inc(x)
+z = add(y, 10)
+```   
+
+is representable like:
+
+
+```python
+dsk = {'x': 1,
```
Comment (martindurant) [inline README.md]: Isn't this a bit detailed for the very first thing we tell potential users?

**Comment by jcrist** [comment]
Not commenting on the content, but this should be restructured text not markdown, as pypi doesn't render markdown readme files.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+![alt text](https://github.com/dask/dask/blob/master/docs/source/_static/dask-simple.png "A simple dask dictionary")
+
+where the computation of `z` is parallelizable
+
+
+```python
+
+from dask import get
+
+z = get(dsk, 'z')
```
Comment (martindurant) [inline README.md]: dask.get is *not* normally what people use

File: README.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+```
+
+
+# Collections and Schedulers
+
+To handle more complex cases there are a variaty of abstractions and schedulers available.
```
Comment (martindurant) [inline README.md]: Really for ALL cases. We haven't even mentioned yet that dask does bigger-than-memory and distributed computation.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+```
+
+## Distributed 
+
+The **concurrent.futures** interface provides general submission of custom
+tasks:
+
+```python
+from dask.distributed import Client
+client = Client('scheduler:port')
```
Comment (martindurant) [inline README.md]: just Client() for the common beginner case.

**Comment by kszucs** [comment]
Do You have pageview statistics? I'm curious of pypi vs github stats.

We can convert with pandoc like here: https://github.com/daskos/mentos/blob/master/setup.py#L5

I think a README should be in markdown, and the documentation in rst.

**Comment by jcrist** [comment]
I'm -1 on requiring pandoc to build the package. There's a cost to added technology, and I don't think restructured text is that bad to read. I also agree with @mrocklin that a readme should be concise and point to larger documentation, so the maintenance cost of requiring pandoc for something small like a readme doesn't seem worth it.

**Comment by kszucs** [comment]
OK then, I'll convert back to rst.

File: README.md (hunk 1 above)
Comment (kszucs) [inline README.md]: IMHO dask is about constructing and executing dags. This part of the documentation gave me the `ahaaa` feeling.

File: README.md (hunk 3 above)
Comment (kszucs) [inline README.md]: Sure, it is just a skeleton. A section like "When is dask a good choice" or "Computations where dask shines" containing a short list: bigger-than-memory, distributed etc. should go to the top of the README. Can You edit this PR?

File: README.md (hunk 4 above)
Comment (kszucs) [inline README.md]: I thought this is the common beginner case for distributed.README

**Comment by mrocklin** [comment]
I think we're just staying with normal documentation here.  Closing.

**Pull request closed without merging**